python/strings_udf/strings_udf/_lib/cpp/strings_udf.pxd

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.
```

Comment:
(Couldn't comment on empty files) Do we need all four of the files:

- `strings_udf/_lib/__init__.pxd` (unused?)
- `strings_udf/_lib/__init__.py` (used to make the package needed for `from strings_udf._lib.cudf_jit_udf import to_string_view_array`)
- `strings_udf/_lib/cpp/__init__.pxd` (used in `from strings_udf._lib.cpp.strings_udf cimport (...)`)
- `strings_udf/_lib/cpp/__init__.py` (unused?)

It strikes me as odd that we should need to be able to `import` from the `cpp` directory (`_lib/cpp/__init__.py`) and `cimport` from the root `_lib/__init__.pxd`. I don't see where these imports are needed.

Reply:
Pruned these files down here